### DISCUSSION TITLE
Self hosting Supabase Studio in AWS

### DISCUSSION THREAD

**Discussion by kuyazee** [supabase/supabase#13062] [Questions]
Hello, I'm looking into self hosting Supabase on AWS and I found [this solution](https://github.com/supabase-community/supabase-on-aws). This however is extremely expensive due to needing to use Fargate.

I have tested out locally that I can host the whole thing on my machine and think a single EC2 instance should be enough.

I am having trouble however on how to actually host Supabase Studio (the web dashboard). Is this hostable in S3 or should I use something else?

**Comment by organicnz** [comment]
Have you tried this [solution](https://docs.digitalocean.com/developer-center/hosting-supabase-on-digitalocean/) on DigitalOcean by any chance? Self-hosting on DO should be much easier IMO :)

**Reply by kuyazee** [reply]
I found that as well, but at the moment prefer AWS
